projectdo

```diff
@@ -287,6 +287,20 @@ try_latex() {
   fi
 }
 
+# Any / Generic
+
+try_build_script() {
+  if [ "$ACTION" = build ]; then
+    if [ -f build.sh ]; then
+      execute "sh -c build.sh"
```

Comment:
Could this be `./build.sh`? That could work better if the script is a Bash script and the user is running the Bash shell.

Reply:
In general `sh script.sh` will execute `sh` and have it run `script.sh`. When doing `./script.sh` the shebang in the top of the script determines what program the script is run through. This makes a difference if the shebang does not specify `sh`. For instance if the shebang is `#!/bin/bash`, then the script should be run with bash. In theory the script could also be written in Python or JavaScript or something else (though it wouldn't make sense with the `.sh` extension).

What I wrote before was slightly inaccurate, the users shell doesn't matter. Running a Bash specific script with `./script.sh` works no matter what the users shell is.